Patch 1 (#2)
oceanhackweek/ohw-helpdesk: README.md

# ohw-helpdesk
OceanHackWeek Q&amp;As

## Purpose

> A place for asynchronous help on data and software in the ocean sciences community. 

Reviewer: ```suggestion
A place for asynchronous help on data and software in the ocean sciences community. 
We plan to start scheduling roughly monthly drop in sessions where we can worth through these questions together.
```

Author: Is this repo taking the place of those meetings? I wonder if it would be better to use this as a mechanism to gather engagement and schedule ad-hoc times to discuss topics of interest?

Reviewer: At last discussion we were still planning on starting those meetings up in the new year once everyone got through the holiday craziness. 

I personally really don't want to have an expectation that we will be meeting ad-hoc for office hours. 

It reduces the chance of building any community because unless you are constantly tuned in you have no idea when they are going to happen, so people don't get a chance to tune in to help answer or see how to work through other peoples questions. 

For the organizers it means that there is some pressure to be able to hop on a call to answer questions at any time. 

Finally by saying 'here is the time each month we will solve questions', we can help people asynchronously craft and refine their questions beforehand, and pull up useful reference material, so that we have a pool of answerable questions to start with during the synchronous meetings.

Author: Sounds good. Someone will probably have to take ownership and coordinate those meetings... Sounds like it's on you @abkfenris for now.

## Scope
* Oceanographic data
* R or Python programming languages

## How to use this repository?
* Search the repository for the question/topic of interest.

> * If nothing exists, add a ticket to describe your topic and include a Minimal Reproducible Example (MRE).

Reviewer: ```suggestion
* If nothing exists, add an issue to describe your topic and include a Minimal Reproducible Example (MRE).
```

>   * Add MRE to `datasets/` via PR.

Reviewer: ```suggestion
  * If a MRE is more unwieldy than can fit in the body of an issue, add MRE to `datasets/` via PR.
```

> * Iterate and document wit the community.

Reviewer: ```suggestion
* Iterate and document with the community.
```

## Purpose
A place for asynchronous help on data and software in the ocean sciences community. 

> We plan to start scheduling roughly monthly drop in sessions where we can worth through these questions together.

Reviewer: ```suggestion
We plan to start scheduling monthly sessions where we can work through these questions together.
```

Tweaking the phrasing while fixing a spelling error that I made earlier.
